vllm/attention/backends/flash_attn.py

```diff
@@ -466,6 +484,7 @@ def build(self, seq_lens: List[int], query_lens: List[int],
             num_prefill_tokens=self.num_prefill_tokens,
             num_decode_tokens=num_decode_tokens,
             seq_lens=seq_lens,
+            multi_modal_placeholder_maps=placeholder_maps,
```

Comment:
What code part is using this added parameter here? Ditto for all other attention changes.

Reply:
Any model that uses `merge_multimodal_embeddings_from_map` -- this change only adds it for Ultravox but that's really only to limit scope. In time I'd expect any multi-modal model that merges multi-modal and text embeddings to use `merge_multimodal_embeddings_from_map` instead of `merge_multimodal_embeddings` (barring any tradeoffs I might be neglecting).